tests/test_odm2/test_readservice.py

```diff
@@ -183,8 +185,11 @@ def test_getRelatedModelsByCode(self):
         assert len(resapi) == 0
 
         # test invalid argument
-        resapi = self.reader.getRelatedModels(code = models.ActionBy)
-        assert resapi is None
+        resapi = self.reader.getRelatedModels(code = 234123)
```

Comment:
If either an empty list or `None` is expected, maybe due to different versions of the dependencies; unclear on the reason, you can use assert not `resapi` there. (sorry for the dummy `a`).

It means that you are expecting a `NULL` value, may it be an empty list or `None`.